android/build.gradle

```
@@ -7,8 +7,10 @@ buildscript {
             google()
             jcenter()
         }
+        def buildGradleVersion = ext.has('buildGradlePluginVersion') ? ext.get('buildGradlePluginVersion') : '3.5.1'
```

```suggestion
        def buildGradleVersion = ext.has('buildGradlePluginVersion') ? ext.get('buildGradlePluginVersion') : '4.1.2'
```

should be current if we are changing it

---

applies any version, because user have ability to specify his buildGradlePluginVersion if he wants